### ISSUE TITLE
bad use of margins

### ISSUE THREAD

**Issue by Walidmash** [facg3/todo-list#17]
I dont know why you need all of these margins, if you really need top, left, right margins at the same time, you can write it in one command like
```margin: 50px 50px 0 50px;```, which is ```top, right, bottom, left``` 
you can read more of how to use [margins](https://developer.mozilla.org/en-US/docs/Web/CSS/margin) from MDN:
https://developer.mozilla.org/en-US/docs/Web/CSS/margin

![screenshot from 2017-11-18 06-31-39](https://user-images.githubusercontent.com/20396442/32977775-be25ba6a-cc3c-11e7-9647-b196477fce79.png)

**Comment by marwa7med** [comment]
yes you sure but sometime we need defferent margin so we determine the direction

**Comment by Walidmash** [comment]
oh, I forgot to upload the picture, 
edited...
